src/addons/transitions/ReactCSSTransitionGroupChild.js

```diff
@@ -85,7 +87,11 @@ var ReactCSSTransitionGroupChild = React.createClass({
     CSSCore.addClass(node, className);
 
     // Need to do this to actually trigger a transition.
-    this.queueClass(activeClassName);
+    if ( REQUEST_ANIMATION_FRAME ) {
+      REQUEST_ANIMATION_FRAME(this.queueClass.bind(this, activeClassName));
```

Comment:
I have observed the same increase in event firing consistency with a direct call to queueClass and queueClass modified as such (POC purposes only):
![screen shot 2015-07-22 at 12 30 14 pm](https://cloud.githubusercontent.com/assets/4976075/8833768/aefe62c0-306d-11e5-8ed4-0fa27e80bcec.png)